convert2rhel/redhatrelease.py

```diff
@@ -49,35 +49,42 @@ def get_system_release_content():
         loggerinst.critical("%s\n%s file is essential for running this tool." % (err, filepath))
 
 
-class YumConf:
-    _yum_conf_path = "/etc/yum.conf"
+class PkgManagerConf:
+    """
+    Check if the config file of the systems package manager has been modified and if it has then
+    remove those changes before the conversion completes.
+    NOTE: The pkg manager config file path only needs to be set to yum.conf as there is a symlink between yum and dnf.
+    This means that on dnf systems the dnf.conf will be modified.
+    """
+
+    _pkg_manager_conf_path = "/etc/yum.conf" if pkgmanager.TYPE == "yum" else "/etc/dnf/dnf.conf"
```

Comment:
I think this could be also used in the `is_modified()` method. I know, now it's static one, but since I see it's called only from `patch()` I think it's not needed anymore.

Reply:
Yes I was also thinking that but the static part made me reconsider. I'll remove the static and use attirbute there